features/db.feature

```diff
@@ -326,3 +326,19 @@ Feature: Perform database operations
       """
       latin1_spanish_ci
       """
+
+  Scenario: Running an UPDATE query should return the number of affected rows
+    Given a WP install
+    When I run `wp db query "UPDATE wp_users SET user_status = 1 WHERE ID = 1"`
+    Then STDOUT should contain:
+      """
+      Query succeeded. Rows affected: 1
+      """
+
+  Scenario: Running an DELETE query should return the number of affected rows
+    Given a WP install
+    When I run `wp db query "DELETE FROM wp_users WHERE ID = 1"`
+    Then STDOUT should contain:
+      """
+      Query succeeded. Rows affected: 1
+      """
```

Comment:
I don't think we need two Scenarios for this. Each one spins up its own WordPress install but this seems like it can all be done in the same Scenario to avoid that. See all of the tests above that run in a single Scenario block. The single name of the Scenario can be something like `Scenario: Row modifying queries should return the number of affected rows` 

Additionally, maybe one more test to verify it is not adding these for SELECT. Untested, but something like

```
    When I run `wp db query "SELECT FROM wp_users WHERE ID = 1"`
    Then STDOUT should not contain:
      """
      Rows affected
      """
```
That test can come before deleting the user